pymc4/gp/gp.py

```diff
@@ -250,13 +296,13 @@ def conditional(
         >>> gp = pm.gp.LatentGP(cov_fn=cov_fn)
         >>> @pm.model
         ... def gp_model():
-        ...     f = yield gp.prior('f', X)
-        ...     fcond = yield gp.conditional('fcond', Xnew, given={'f': f, 'X': X})
+        ...     f = yield gp.prior('f', X, jitter=1e-3)
+        ...     fcond = yield gp.conditional('fcond', Xnew, given={'f': f, 'X': X}, jitter=1e-3)
```

Comment:
Is the `jitter` here related to the one mentionned by the docstrings? If yes, I'm guessing this should be 1e-4 or 1e-6 instead of 1e-3, shouldn't it?

Reply:
The docs kept failing with `1e-4` jitter so I had to change it to `1e-4`. Though this isn't related to the default value of `jitter` argument. This is just an independent example using a particular value of `jitter`. So, default values are still `1e-4` for `float32` datatype and `1e-6` for `float64` datatypes. Hope this explains it all...

Reply:
Ah I see, thanks!